default/buildings.txt

```diff
@@ -625,7 +625,7 @@ BuildingType
             stackinggroup = "INDUSTRY_CENTER_STACK"
             effects = SetTargetIndustry value = Value + Target.Population * 3 * [[INDUSTRY_PER_POP]]
     ]
-    icon = ""
+    icon = "icons/tech/industrial_centre_ii.png"
```

Comment:
I mean that I want to have 2 pull requests for this two features.   
I mean that then you see diff for pull request and see that you have two features you will not create it.
Please forget **rebasing** and **squashing** words for a while.  You don't feel comfort on basic git operation. 

Process of adding feature is next:
- update you master to actual state
- create new branch (you did not do it then apply changes form Sloth thats why you got mix)
- make your feature (or merge changes from other branch) and dont forget to merge actual master in during development
- Push (all rebasing and squashing can be done before it step)
- Make pr (Always check diff before creating pr!)